<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imitator</class>
 <widget class="QMainWindow" name="Imitator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imitator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="linearsButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Линейные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anglesButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Угловые</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overloadsButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Перегрузки</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="xPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="zPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="yPlot" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="vPlot" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_stopButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>691</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="gammaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="nzPlot" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="thetaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="nxPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="psiPlot" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="nyPlot" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>711</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="connectionRadioButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>690</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QWidget" name="valuesWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1074</width>
      <height>800</height>
     </rect>
    </property>
    <widget class="QLabel" name="modelParamsLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Начальные значения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="desiredlParamsLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Желаемые значения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="vLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Абсолютная скорость (м/с)</string>
     </property>
    </widget>
    <widget class="QLabel" name="xLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Координата X (м)</string>
     </property>
    </widget>
    <widget class="QLabel" name="yLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Координата Y (м)</string>
     </property>
    </widget>
    <widget class="QLabel" name="zLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Координата Z (м)</string>
     </property>
    </widget>
    <widget class="QLabel" name="thetaLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Угол Θ (град)</string>
     </property>
    </widget>
    <widget class="QLabel" name="psiLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Угол Ψ (град)</string>
     </property>
    </widget>
    <widget class="QLabel" name="gammaLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Угол γ (град)</string>
     </property>
    </widget>
    <widget class="QLabel" name="nxLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Перегрузка nx</string>
     </property>
    </widget>
    <widget class="QLabel" name="nyLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Перегрузка ny</string>
     </property>
    </widget>
    <widget class="QLabel" name="nzLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Перегрузка nz</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="thetaLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="psiLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gammaLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nxLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nyLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>390</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nzLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setParamsButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>480</y>
       <width>151</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>Задать начальные
значения</string>
     </property>
    </widget>
    <widget class="QLabel" name="vDesLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Желаемая абс. скорость (м/с)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vDesLineEdit">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>35</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yDesLineEdit">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="yDesLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Координата Y (м)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="psiDesLineEdit">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="psiDesLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Угол Ψ (град)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setDesiredButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>200</y>
       <width>151</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>Задать желаемые
значения</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="showGraphics"/>
    <addaction name="showValues"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="showGraphics">
   <property name="text">
    <string>Графики</string>
   </property>
  </action>
  <action name="showValues">
   <property name="text">
    <string>Значения</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
